datasets/streaming/error_aggregates/intro.md

```diff
@@ -0,0 +1,17 @@
+The `error_aggregates` table represents counts of errors counted from main and crash
+pings, aggregated every 5 minutes.
+
+#### Contents
+
+The `error_aggregates` table contains counts of raw counts of various error measures,
+aggregated across each unique set of dimensions (for example: channel, operating system).
+You can get an aggregated count for any particular set of dimensions by summing using SQL.
+
+#### Accessing the data
+
+You can access the data via re:dash. Choose `Athena` and then select the
```

Comment:
When v2 lands it won't be available in Athena any longer, what with the hll columns.

Reply:
we need to think about whether that trade-off is worth it given the utility of the hll columns, as the performance of presto is pretty awful as discussed. anyway, i'll keep this as-is for now.